packages/yara/yara.0.1/opam

```diff
@@ -0,0 +1,30 @@
+# -*- conf -*-
+opam-version: "1.2"
+name: "yara"
+version: "0.1"
+maintainer: "Anton Kochkov <anton.kochkov@gmail.com>"
+author: "Anton Kochkov <anton.kochkov@gmail.com>"
+homepage: "https://github.com/XVilka/yara-ocaml"
+bug-reports: "https://github.com/XVilka/yara-ocaml/issues"
+license: "MIT"
+dev-repo: "https://github.com/XVilka/yara-ocaml.git"
+build: [
+	["jbuilder" "subst" "-n" name] {pinned}
+	["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+	"jbuilder" {build}
+	"ppx_deriving" {build}
```

Comment:
This can't really be right. You're almost certainly depending on some runtime component in `ppx_deriving` so it shouldn't be a build dependency.

You should list `ocaml-migrate-parsetree` as a build dependency however.